vault/server.sls

```diff
@@ -109,4 +109,8 @@ vault:
       {% endif %}
       {%- if vault.tls_cert_file_content is defined %}
       - file: install_tls_cert_file
-      {% endif %}
\ No newline at end of file
+      {% endif %}
+
+vault-init:
```

Comment:
is this state executed every time ? I don't see a vault init check.

Reply:
Will not you have a conflict with the vault formula ?

Reply:
We might merge issue with up stream formula if we do this way. One way could be to extend it. Salt support https://docs.saltstack.com/en/latest/ref/states/extend.html

Reply:
this does not do a vault init. It has to be done manually because this is the step that would return a root token and unseal keys.. This bit is to just echo out to remind the manual step required.

Reply:
Ok I see. Please check how to extend formula it might avoid us maintenance nightmare.

Reply:
Good idea. Let me check this out..